rasa/core/tracker_store.py

```diff
@@ -917,33 +917,44 @@ def get_db_url(
         )
 
     def _create_database_and_update_engine(self, db: Text, engine_url: "URL"):
-        """Create databse `db` and update engine to reflect the updated `engine_url`."""
-
+        """Creates database `db` and updates engine accordingly."""
         from sqlalchemy import create_engine
 
+        if not self.engine.dialect.name == "postgresql":
```

Comment:
we could also use https://sqlalchemy-utils.readthedocs.io/en/latest/_modules/sqlalchemy_utils/functions/database.html#create_database but that only supports mysql in addition which doesn't seem worth to include it as extra library (`login_db`  doesn't make sense for sqlite)